Sources/TSCBasic/misc.swift

```diff
@@ -224,6 +224,11 @@ public func lookupExecutablePath(
     guard let value = value, !value.isEmpty else {
         return nil
     }
+#if os(Windows)
+    let isPath = value.contains(":") || value.contains("\\") || value.contains("/")
```

Comment:
I don't know about the `value.contains(":")`. It should be explicit at index 1 for the drive, or it may be after the file extension as an ADS indicator.

Reply:
I will update the naming. This shouldn’t be `isPath`, but `isNotFilename` (will use `isFilename` instead).